validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -271,16 +271,15 @@ public static ValidatorClientService create(
             })
         .exceptionally(
             error -> {
-              ExceptionUtil.getCause(error, InvalidConfigurationException.class)
-                  .ifPresentOrElse(
-                      cause -> LOG.error(cause.getMessage()),
-                      () ->
-                          LOG.error(
-                              "Error was encountered during validator client service start up.",
-                              error));
+              LOG.error("Error was encountered during validator client service start up.", error);
+              if (ExceptionUtil.hasCause(error, ValidatorLoaderException.class)) {
+                STATUS_LOG.failedToLoadValidatorKey(ExceptionUtil.getRootCauseMessage(error));
+              } else {
+                STATUS_LOG.errorDuringValidatorClientInitialization(
+                    ExceptionUtil.getRootCauseMessage(error));
+              }
 
-              LOG.error(
-                  "Unable to initialize validator keys, please manually correct errors and try again.");
+              STATUS_LOG.failedToStartValidatorClient();
```

Comment:
i'd be ok with this change so that its on the console, but im not convinced about the 274-280 change, i think it was better previously in terms of the content delivered.

Reply:
I want to clearly separate keys loading and other exceptions.